index.d.ts

```diff
@@ -20,13 +20,21 @@ export interface Flag<Type extends FlagType, Default> {
 	readonly isMultiple?: boolean;
 }
 
+type ImportMeta = {url: string};
```

Comment:
```suggestion
type ImportMeta = typeof import.meta;
```

?

Reply:
It looks like it's not correct grammar.

Reply:
How about use `Record<string, unknown> & {url: string}` instead?

Reply:
We could use `ImportMeta` directly.

The type definitions is here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/globals.d.ts#L29